Panic on datalog with deny policy

Using the next datalog 
```golang
auth, err := parser.FromStringAuthorizer(`
	deny if resource($r), $r.contains("T");
`)
if err != nil {
	t.Fatalf("cannot compute Authorizer: %s", err.Error())
}
```

result in 
```
--- FAIL: Test_debug (0.00s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xd41d5c]

goroutine 35 [running]:
testing.tRunner.func1.2({0xefa720, 0x187d230})
	/usr/lib/go/src/testing/testing.go:1632 +0x230
testing.tRunner.func1()
	/usr/lib/go/src/testing/testing.go:1635 +0x35e
panic({0xefa720?, 0x187d230?})
	/usr/lib/go/src/runtime/panic.go:785 +0x132
github.com/biscuit-auth/biscuit-go/v2/parser.(*Policy).ToBiscuit(0x0?, 0x0)
	/vendor/github.com/biscuit-auth/biscuit-go/v2/parser/grammar.go:671 +0x3c
github.com/biscuit-auth/biscuit-go/v2/parser.(*Authorizer).ToBiscuit(0xc000165a40?, 0x0)
	/vendor/github.com/biscuit-auth/biscuit-go/v2/parser/grammar.go:158 +0x135
github.com/biscuit-auth/biscuit-go/v2/parser.(*parser).Authorizer(0x20?, {0x106e94e?, 0x0?}, 0x0)
	/vendor/github.com/biscuit-auth/biscuit-go/v2/parser/parser.go:205 +0x7b
github.com/biscuit-auth/biscuit-go/v2/parser.FromStringAuthorizerWithParams({0x106e94e, 0x2c}, 0x0)
	/vendor/github.com/biscuit-auth/biscuit-go/v2/parser/parser.go:328 +0x9b
github.com/biscuit-auth/biscuit-go/v2/parser.FromStringAuthorizer(...)
	/github.com/biscuit-auth/biscuit-go/v2/parser/parser.go:292
opentelemetry/pkg.Test_debug(0xc000414ea0)
	/pkg/biscuit_test.go:68 +0x846
testing.tRunner(0xc000414ea0, 0x10bd298)
	/usr/lib/go/src/testing/testing.go:1690 +0xf4
created by testing.(*T).Run in goroutine 1
	/usr/lib/go/src/testing/testing.go:1743 +0x390
```

--- a/parser/grammar.go
+++ b/parser/grammar.go
@@ -668,7 +668,7 @@ func (p *Policy) ToBiscuit(parameters ParametersMap) (*biscuit.Policy, error) {
 		}
 	case p.Deny != nil:
 		{
-			parsedQueries = p.Allow.Queries
+			parsedQueries = p.Deny.Queries
 			kind = biscuit.PolicyKindDeny
 			break
 		}

